July 14 poll drafts (#33)
sky-ecosystem/polls: 2025/2025-07-14-mainnet-spark-usds-vault-onboard-pt-spk-usds-farm.md

---

> title: Spark USDS Vault Mainnet - Onboard PT-SPK-USDS Farm - July 14, 2025

Reviewer: Make it clearer that this is a new vault

```suggestion
title: Spark Liquidity Layer Mainnet - Deploy Spark USDS Morpho Vault and Onboard PT-SPK-USDS Farm - July 14, 2025
```

> title: Spark Liquidity Layer Mainnet - Deploy Spark USDS Morpho Vault and Onboard PT-SPK-USDS Farm - July 14, 2025

Reviewer: Disagree on this one, it's a Spark DAI Morpho Vault has been the preferred convention for the Dai product so we should keep this consistent for the USDS product in polls and executive votes.

> title: Spark Liquidity Layer Mainnet - Deploy Spark USDS Morpho Vault and Onboard PT-SPK-USDS Farm - July 14, 2025

Reviewer 2: In this case, the second header is incorrect. I noticed both use different versions, so I suggested a change for this one

> summary: Signal your support or opposition to onboarding PT-USDS-24Sep2025 to the Spark USDS Vault.

Reviewer: ```suggestion
summary: Signal your support or opposition to deploying a new Spark USDS Morpho Vault and onboarding PT-USDS-24Sep2025.
```

discussion_link: https://forum.sky.money/t/july-24-2025-proposed-changes-to-spark-for-upcoming-spell/26796
parameters:
    input_format: single-choice
    victory_conditions:
        - { type : plurality }
    result_display: single-vote-breakdown
version: v2.0.0
options:
   0: Abstain
   1: Yes
   2: No
start_date: 2025-07-14T16:00:00
end_date: 2025-07-17T16:00:00
---

> # Poll: Spark USDS Vault Mainnet - Onboard PT-SPK-USDS Farm - July 14, 2025

Reviewer: ```suggestion
# Poll: Spark Liquidity Layer Mainnet - Deploy Spark Morpho USDS Vault and Onboard PT-SPK-USDS Farm - July 14, 2025
```

> # Poll: Spark Liquidity Layer Mainnet - Deploy Spark Morpho USDS Vault and Onboard PT-SPK-USDS Farm - July 14, 2025

Reviewer 2: ```suggestion
# Poll: Spark Liquidity Layer Mainnet - Deploy Spark USDS Morpho Vault and Onboard PT-SPK-USDS Farm - July 14, 2025
```

The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Stability Facilitators. This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9.1_Operational_Weekly_Cycle-b189fa17-57a9-4d4e-9780-0ce4efd94211%7C0db30308) will be active for three days beginning on Monday, July 14 at 16:00 UTC.

**This is a binary vote.**

- **You may vote for a single option.**
- **You should vote for the option which you prefer.**
- **If you would accept either option, you should vote 'Abstain'.**

## Review

> The community can vote in this poll to express support or opposition to onboarding the PT-SPK-USDS Farm to the Spark USDS Vault with the following parameters:

Reviewer: ```suggestion
The community can vote in this poll to express support or opposition to deploying a Spark USDS Morpho Vaul and onboarding the PT-SPK-USDS Farm with the following parameters:
```

> The community can vote in this poll to express support or opposition to deploying a Spark USDS Morpho Vaul and onboarding the PT-SPK-USDS Farm with the following parameters:

Reviewer: typo

```suggestion
The community can vote in this poll to express support or opposition to deploying a Spark USDS Morpho Vault and onboarding the PT-SPK-USDS Farm with the following parameters:
```
